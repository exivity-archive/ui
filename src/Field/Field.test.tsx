/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'

import { Field } from '.'

test('renders default Field', () => {
  const component = renderer.create(<Field />)
  expect(component.toJSON()).toMatchSnapshot()
})

test('renders default Field.Container', () => {
  const component = renderer.create(<Field.Container />)
  expect(component.toJSON()).toMatchSnapshot()
})
