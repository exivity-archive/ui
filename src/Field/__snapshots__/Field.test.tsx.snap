// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default Field 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: unset;
  -webkit-box-align: unset;
  -ms-flex-align: unset;
  align-items: unset;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  white-space: unset;
}

.c0:not(:last-child) {
  margin-bottom: 1.25em;
}

.c0 .c1 {
  padding-bottom: 0.625em;
  margin-right: unset;
  -webkit-flex-basis: auto;
  -ms-flex-preferred-size: auto;
  flex-basis: auto;
}

.c0 .c1 > .c1 {
  padding-bottom: unset;
}

<div
  className="c0"
/>
`;
